src/iqm/cirq_iqm/iqm_operation_mapping.py

```diff
@@ -53,6 +63,8 @@ def instruction_to_operation(instr: Instruction) -> Operation:
         args['key'] = instr.args['key']
         args['num_qubits'] = len(instr.qubits)
     qubits = [NamedQid(qubit, dimension=2) for qubit in instr.qubits]
+    if instr.name == 'cc_prx':
```

Comment:
I can't refer to it direcly, but what is the "num_qubits" arg above?

Reply:
This function should be rewritten as an if/elif/else construct where for each operation type we look for the correct args.